gramex/handlers/mlhandler.py

```diff
@@ -450,6 +450,10 @@ def get(self, *path_args, **path_kwargs):
             self.write(json.dumps(params, indent=2))
         elif '_cache' in self.args:
             self.write(self.load_data().to_json(orient='records'))
+        elif 'text' in self.args:
```

Comment:
This condition doesn't check `backend`. Which means, that even if I'm running an sklearn model which has one column named `text`, and I request predictions from this model - this piece of code will run.